src/simulation/m_compute_cbc.fpp

```diff
@@ -28,7 +28,11 @@ contains
         !!      the normal component of velocity is zero at all times,
         !!      while the transverse velocities may be nonzero.
     subroutine s_compute_slip_wall_L(lambda, L, rho, c, mf, dalpha_rho_ds, dpres_ds, dvel_ds, dadv_ds) ! --------------
+#ifdef CRAY_ACC_WAR
+        !DIR$ INLINEALWAYS s_compute_slip_wall_L
```

Comment:
is it ok that this statement is indented? I'm not sure. May need to ask @abbotts

Reply:
nevermind it is fine https://docs.oracle.com/cd/E19957-01/805-4941/6j4m2sobl/index.html